src/ontology/cl-edit.owl

```diff
@@ -6662,9 +6662,10 @@ SubClassOf(obo:CL_0000384 obo:CL_0000630)
 
 # Class: obo:CL_0000385 (prohemocyte (sensu Nematoda and Protostomia))
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/B978-012369493-5.50008-0") obo:IAO_0000115 obo:CL_0000385 "A precursor of mature hemocytes.")
 AnnotationAssertion(obo:RO_0002161 obo:CL_0000385 obo:NCBITaxon_33511)
 AnnotationAssertion(rdfs:label obo:CL_0000385 "prohemocyte (sensu Nematoda and Protostomia)")
-SubClassOf(obo:CL_0000385 obo:CL_0000390)
+SubClassOf(obo:CL_0000385 obo:CL_0000988)
```

Comment:
Should this be CL:0008001 'hematopoietic precursor cell' instead'?
dbxref behind paywall.

Reply:
That’d be more precise indeed. Thanks!